classes/fields/file.php

```diff
@@ -544,6 +544,7 @@ public function build_dfv_field_options( $options, $args ) {
 					'pod'      => $pod_id,
 					'field'    => $field_id,
 					'item_id'  => $item_id,
+					'post_id'  => $item_id,
```

Comment:
@sc0ttkclark 
This part is misleading since this could be a tax ID or user ID as well. It's the Pod item ID, not necessarily a Post ID.

Reply:
Also, I think if this is preferable then it should get a comment about backwards compatibility so we know why this is still here in the future.

Reply:
Resolved in https://github.com/pods-framework/pods/pull/6151